src/metabase/api/card.clj

```diff
@@ -183,8 +184,11 @@
    display                su/NonBlankString
    visualization_settings su/Map
    collection_id          (s/maybe su/IntGreaterThanZero)}
-  ;; check that we have permissions to run the query that we're trying to save
-  (check-403 (perms/set-has-full-permissions-for-set? @*current-user-permissions-set* (card/query-perms-set dataset_query :write)))
+  ;; check that we have permissions to run the query that we're trying to save, skip it for googleanalytics
+  (cond
+    (= ((get-db (dataset_query :database)) :engine) :googleanalytics) (println "googleanalytics card")
+    :else  (check-403 (perms/set-has-full-permissions-for-set? @*current-user-permissions-set* (card/query-perms-set dataset_query :write))))
```

Comment:
Ah!  Will keep that in mind for future. Thanks. I have no experience in clojure. This was all done based upon trial and error. haha